.github/workflows/renovate.yaml

```
@@ -13,7 +13,7 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4
       - name: Self-hosted Renovate
-        uses: renovatebot/github-action@v39.2.4
+        uses: renovatebot/github-action@v40.0.0
```

```suggestion
        uses: renovatebot/github-action@v40.0.2
```
https://github.com/renovatebot/github-action/releases